portfolio/src/main/webapp/script.js

```diff
@@ -63,14 +63,23 @@ function previousExperience() {
   selectedExperience--;
 }
 
+var number = 0;
```

Comment:
I also don't think this variable needs to have a global scope. Think about how you might be able to accomplish this, perhaps by adding a parameter to the `getServerComments` function.

Reply:
This is a good point, thank you, I'm just having some trouble with one aspect. I know if I initialize number inside storeNumberComments(), I could pass it to getServerComments(number) as a parameter. However in my index.html, I have <body onload="getServerComments()" so that when the user posts a new comment, the page re-fetches the comments so that the new one shows up. When this happens, though, the value that was selected in the drop down (for the number of comments to display) is lost because it resets. Since I would now need to go through storeNumberComments() to get to getServerComments() for the onload=, I need to figure out a way to preserve the selected value in the drop down, but I'm having trouble doing this. That is why I had given the variable global scope, but I will continue researching how to do this. If you have any suggestions on how to preserve the value, that'd be great. In the meantime I'll keep working at it!

Reply:
Sorry for that long message, I just wanted to add that I've tried to solve it by using local storage so that when the page reloads, the drop down gets the value that was selected from storage, which is what almost all answers say to do online, but I keep getting errors.